website/client/src/components/groups/membersModal.vue

```diff
@@ -681,9 +687,12 @@ export default {
       if (memberId === this.group.leader || memberId === this.group.leader._id) return false;
       return this.groupIsSubscribed && !(this.group.managers && this.group.managers[memberId]);
     },
-    shouldShowRemoveManager (memberId) {
+    shouldShowRemoveManager () {
+      return true;
```

Comment:
is this expected as part of this PR?

Reply:
oh no, I was testing to see the remove buttons (because of the icons), will be removed